src/js/tabs/vtx.js

```diff
@@ -40,12 +40,22 @@ vtx.isVtxDeviceStatusNotReady = function()
 
 vtx.updateVtxDeviceStatus = function()
 {
-    MSP.send_message(MSPCodes.MSP2_GET_VTX_DEVICE_STATUS, false, false, vtxDeviceStatusReceived);
-
     function vtxDeviceStatusReceived()
     {
         $("#vtx_type_description").text(TABS.vtx.getVtxTypeString());
     }
+
+    function vtxDeviceStatusReady()
+    {
+        const vtxReady_e = $('.VTX_info span.vtx_device_ready');
+
+        // update device ready state
+        vtxReady_e.text((FC.VTX_CONFIG.vtx_device_ready) ? i18n.getMessage('vtxReadyTrue') : i18n.getMessage('vtxReadyFalse'));
```

Comment:
```suggestion
        vtxReady_e.text(FC.VTX_CONFIG.vtx_device_ready ? i18n.getMessage('vtxReadyTrue') : i18n.getMessage('vtxReadyFalse'));
```

Reply:
This is still unresolved :stuck_out_tongue:

Reply:
argh